src/main/java/mate/academy/onlinebookstoreproject/model/Book.java

```diff
@@ -28,6 +32,8 @@ public class Book {
     private BigDecimal price;
     private String description;
     private String coverImage;
+    @Column(nullable = false)
+    private boolean isDeleted = false;
 
     public Book() {

```

Comment:
Why do you need this constructor if you use Data?

Reply:
Makes sense, but not critical.